test/e2e/validation_test.go

```diff
@@ -464,6 +464,11 @@ func testExpectedServicesRunning(t *testing.T) {
 	}
 }
 
+// testKubeletCARotation test the kubelet CA rotation
+func testKubeletCARotation(t *testing.T) {
+	t.Skip("Not implemented yet")
```

Comment:
Is there going to be a different ticket to track this one?

Reply:
Included in [e163351 ](https://github.com/openshift/windows-machine-config-operator/commit/e163351929fa3facd6ca7be1ffc02d722204b353)